lib/moneta/weak.rb

```diff
@@ -28,4 +28,19 @@ def initialize(adapter, options = {})
       super
     end
   end
+
+  # Adds weak key enumeration support to the underlying store
+  #
+  # @note The each_key method might not iterate over the whole set of keys or efficiently enough (this is meant by weak)
```

Comment:
This note needs to be more detailed about the circumstances where this proxy will yield incorrect results, so that users know what they are in for.  Namely:

- Persistent stores,
- Backends accessed by more than one store object (multi-process, etc.),
- Concurrency?